src/util/ConfigStore.cc

```diff
@@ -95,25 +95,33 @@ std::string ConfigStore::get(std::string const& key,
     }
 }
 
+
 int ConfigStore::getInt(std::string const& key, int const& defaultValue) const {
+    try {
+        return getIntRequired(key);
+    } catch (util::KeyNotFoundError& e) {
```

Comment:
Catch by const reference. I'd generally try to avoid using exceptions for non-exceptional cases in C++.